signer/src/bitcoin/packaging.rs

```diff
@@ -64,16 +64,15 @@ where
     I: IntoIterator<Item = T>,
     T: Weighted,
 {
-    // This is a variant of the Best-Fit-Decreasing algorithm, so we sort
-    // by "weight" decreasing. We use the votes against as the weight, but
-    // vsize is a reasonable weight metric as well.
-    let mut item_vec: Vec<(u32, T)> = items
+    // We might want to sort the items, and previously the was sorted by votes.
+    // However, it can allow malicious signer to speedup/slowdown requests execution,
+    // so it is not so good key to sort with. Some better keys can be time or something
+    // like this, but it require a bit more work so for now we just keep it unsorted.
+    let item_vec: Vec<(u32, T)> = items
```

Comment:
I would remove the comment entirely. Mentioning the sorting feels out of place, it's not like the items look like they need to be sorted. In fact, we could delete the `item_vec` variable and update https://github.com/stacks-sbtc/sbtc/pull/1664/files#diff-6d30e6b110dfd483e31bcd5e4715895aa5ab253686a1e5858ff63781fddabbc9R82.

Reply:
[fd10811](https://github.com/stacks-sbtc/sbtc/pull/1664/commits/fd10811e655eb9b2d170565a9b53dbe9cbcae3b4)